pages/form/returning-members.js

```diff
@@ -225,35 +200,34 @@ const ReturningMembersForm = () => {
               onChange={handleInputChange}
               hasError={hasError['fullName']}
               errorText="Please enter your full name."
+              placeholder='First Name and Last Name'
             />
             <FieldSection
-              title="This upcoming term, I will be on my"
+              title="This upcoming term, I will be in my"
```

Comment:
Should we have this in `registration.js` as well?

Reply:
That's a question I will ask PMs/Directors next term.

In the current Google form, We already ask them to list **future terms** they will be on Waterloop team. 

Currently, I won't add this to `register.js` for this MR. 

Later on, these Teamhub forms with have editable questions/options. So, admins can add this question to the form if they want.

Reply:
"Term" in this context is different though no? This line is for like "1A Co-Op", "2A Study", etc., while the "future terms" question you're referring to is for like F22, W23, etc. I was asking if maybe we should just have the user fill out their term from the registration form, like in the returning members form, that way they don't have to go into their profile and change it.

Reply:
Yeah, it does make sense to me to include this question in the Registration form. I will make this change tomorrow and push it.